pilot/pkg/model/telemetry.go

```diff
@@ -1052,3 +1056,56 @@ func generateStatsConfig(class networking.ListenerClass, metricsCfg telemetryFil
 func disableHostHeaderFallback(class networking.ListenerClass) bool {
 	return class == networking.ListenerClassSidecarInbound || class == networking.ListenerClassGateway
 }
+
+// resourceAttributes return semantic attributes follow OpenTelemetry specification
+// see more details here: https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/
+func resourceAttributes(proxy *Proxy) *otlpcommon.KeyValueList {
+	podName, _ := extractProxyID(proxy)
+	canonicalName := proxy.Labels[IstioCanonicalServiceLabelName]
+	canonicalRevision := proxy.Labels[IstioCanonicalServiceRevisionLabelName]
+	return &otlpcommon.KeyValueList{
+		Values: []*otlpcommon.KeyValue{
+			otelKeyValue(otelsemconv.K8SClusterNameKey, proxy.Metadata.ClusterID.String()),
+			otelKeyValue(otelsemconv.K8SNamespaceNameKey, proxy.ConfigNamespace),
+			otelKeyValue(otelsemconv.K8SDeploymentNameKey, proxy.Metadata.WorkloadName),
```

Comment:
Deployment may not be correct, we need to correctly identify the parent resource type.

Reply:
`Owners and Dependents` lost after convert to `Proxy`